common-containers/src/clients/ApiClient.ts

```diff
@@ -143,6 +147,81 @@ export class ApiClient implements Injectable {
 			}
 	}
 
+	private getGasFees = async (network: number) => {
+        const response = await fetch(`https://api-gateway-v1.stage.svcs.ferrumnetwork.io/api/v1/gasFees/${network}`)
```

Comment:
this should be prod URL, We should let this PR merge and raise a fix for this, as this PR is already deployed on prod